tools/dependencies/known-dependencies.txt

```diff
@@ -214,13 +217,13 @@ hadoop-yarn-server-common-2.6.5.jar
 hbase-annotations-2.0.0.jar
 hbase-client-2.0.0.jar
 hbase-client-2.1.0.jar
-hbase-common-2.0.0.jar
 hbase-common-2.0.0-tests.jar
+hbase-common-2.0.0.jar
 hbase-common-2.1.0.jar
-hbase-hadoop2-compat-2.0.0.jar
-hbase-hadoop2-compat-2.1.0.jar
 hbase-hadoop-compat-2.0.0.jar
 hbase-hadoop-compat-2.1.0.jar
+hbase-hadoop2-compat-2.0.0.jar
```

Comment:
Why is the new Jar introduced, are you missing something?

Reply:
Hi @CalvinKirs, there is no new Jar, just the positions have been switched.

Reply:
Later, I may raise another PR to ensure the items in `known-dependencies.txt` is sorted. What do you think

Reply:
@CalvinKirs It's done, FYI, https://github.com/apache/incubator-seatunnel/pull/1286